charts/v2.1/cray-hms-scsd/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: "cray-hms-scsd"
+version: 2.0.2
```

Comment:
This needs to be updated to version: 2.1.0.

Reply:
Updated